### PR TITLE
ACP-4516: Integrated algolia events

### PR DESCRIPTION
**Pull request by vol4onok**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/ACP-4516

###### Optional

- Core PR: URL_HERE

###### Change log

{Relevant changes for project level go here.}

### CODE REVIEW

File: src/Pyz/Yves/CheckoutPage/Theme/default/templates/page-layout-order-submitted/page-layout-order-submitted.twig [hunk 1]
```diff
@@ -26,3 +26,22 @@
         {% endif %}
     </div>
 {% endblock %}
+
```
Comment (matweew) [inline src/Pyz/Yves/CheckoutPage/Theme/default/templates/page-layout-order-submitted/page-layout-order-submitted.twig:29]: if this page will be extended from `{% extends template('page-layout-order-submitted.', '@SprykerShop:CheckoutPage') %}` can we remove this copy-paste block?

File: src/Pyz/Yves/HomePage/Theme/default/views/home/home.twig [hunk 2]
```diff
@@ -28,3 +28,15 @@
         </main>
     </div>
 {% endblock %}
+
+{% block eventTracker %}
```
Comment (matweew) [inline src/Pyz/Yves/HomePage/Theme/default/views/home/home.twig]: the same Q, can we extend `https://github.com/spryker-shop/home-page/blob/master/src/SprykerShop/Yves/HomePage/Theme/default/views/home/home.twig` and don't include this block

File: src/Pyz/Yves/CheckoutPage/Theme/default/templates/page-layout-order-submitted/page-layout-order-submitted.twig (hunk 1 above)
Comment (supproduction) [reply src/Pyz/Yves/CheckoutPage/Theme/default/templates/page-layout-order-submitted/page-layout-order-submitted.twig:29]: yes we can. But this block is not extendeds from `page-layout-order-submitted`

File: src/Pyz/Yves/HomePage/Theme/default/views/home/home.twig (hunk 2 above)
Comment (supproduction) [reply src/Pyz/Yves/HomePage/Theme/default/views/home/home.twig]: Updated

File: src/Pyz/Yves/CheckoutPage/Theme/default/templates/page-layout-order-submitted/page-layout-order-submitted.twig (hunk 1 above)
Comment (matweew) [reply src/Pyz/Yves/CheckoutPage/Theme/default/templates/page-layout-order-submitted/page-layout-order-submitted.twig:29]: discussed, the change will break the UI.

**Review by matweew — APPROVED** [review]

**Review by tolerants — APPROVED** [review]

**Review by matweew — APPROVED** [review]

**Pull request merged**